packages/sirius-web/backend/sirius-web-sample-application/src/main/java/org/eclipse/sirius/web/sample/services/DomainAttributeServices.java

```diff
@@ -37,10 +48,18 @@ public String getDataType(Attribute attr) {
     public EObject setDataType(Attribute attr, String typeName) {
         var type = DataType.getByName(typeName);
         attr.setType(type);
+        if (type == null) {
+            this.feedbackMessageService.addFeedbackMessage(new Message("Reverting type of %s to the default type %s".formatted(attr.getName(),
+                    this.capitalize(DataType.STRING.getLiteral())), MessageLevel.WARNING));
+        } else {
+            this.feedbackMessageService.addFeedbackMessage(new Message("Set type of %s to %s".formatted(attr.getName(), this.capitalize(type.getLiteral())), MessageLevel.SUCCESS));
+            this.feedbackMessageService.addFeedbackMessage(new Message("You can stack messages", MessageLevel.INFO));
+            this.feedbackMessageService.addFeedbackMessage(new Message("With different level", MessageLevel.WARNING));
+        }
```

Comment:
Except maybe for the "Reverting type to the default" (which can be interesting because until #1994 is fixed the behavior can be confusing), the rest are only useful for a demo/capturing a sample GIF, but should not be commited.

Reply:
Yes, I used it for capturing purpose, I've removed them.